e2e/various-suite/explore.spec.ts

```diff
@@ -1,4 +1,5 @@
 import { e2e } from '@grafana/e2e';
+import { urlUtil } from '../../packages/grafana-data/src/utils/url';
```

Comment:
```suggestion
import { urlUtil } from '@grafana/data';
```

Reply:
@ifrost I tried this and webpack has an issue on a seemingly unrelated file. Have you ever ran into this before? 

`Webpack Compilation Error
Module not found: Error: Can't resolve './FieldConfigOptionsRegistry' in '/Users/kristinadurivage/development/grafana/packages/grafana-data/src/field'`

Reply:
I ended up thinking that it didnt make a lot of sense to bring in that whole library for one utility function. Instead, anticipating we may want to test on URL changes more with cypress, I moved the utility function over and exposed it so it can be used by cypress tests from our e2e package.

Reply:
This works perfectly - I didn't know about that one! Thanks!